packages/appinio_swiper/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/appinioGmbH/flutter_packages
 repository: https://github.com/appinioGmbH/flutter_packages/tree/main/packages/appinio_swiper
 
 environment:
-  sdk: ">=2.15.0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
Dart 3 has a bunch of snazzy features like sealed classes and switch expressions.